glog should not be used directly in Rook

**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
The rook main entry point binary crashes with the following while I'm trying to upgrade the Kubernetes references to 1.13 as described in #2806:
```
/usr/local/bin/rook flag redefined: log_dir
panic: /usr/local/bin/rook flag redefined: log_dir

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc0000ac180, 0x1ed2b40, 0xc0003f6120, 0x1c619e5, 0x7, 0x1cadde7, 0x2f)
	/usr/local/go/src/flag/flag.go:805 +0x529
flag.(*FlagSet).StringVar(0xc0000ac180, 0xc0003f6120, 0x1c619e5, 0x7, 0x0, 0x0, 0x1cadde7, 0x2f)
	/usr/local/go/src/flag/flag.go:708 +0x8a
flag.(*FlagSet).String(0xc0000ac180, 0x1c619e5, 0x7, 0x0, 0x0, 0x1cadde7, 0x2f, 0xc0003f6110)
	/usr/local/go/src/flag/flag.go:721 +0x8b
flag.String(0x1c619e5, 0x7, 0x0, 0x0, 0x1cadde7, 0x2f, 0x0)
	/usr/local/go/src/flag/flag.go:728 +0x69
```

The cause appears to be the EdgeFS operator's single usage of `glog`: https://github.com/rook/rook/blob/master/pkg/operator/edgefs/cluster/utils.go#L24

`glog` should not be used directly by the Rook operators, instead the consistent logger pattern shown here should be used instead: https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/controller.go#L65

**Expected behavior:** Rook binary not to crash/panic

**How to reproduce it (minimal and precise):**
Not a direct/easy repo since this requires updating the Rook dependencies to k8s 1.13, and i'm not sure why that triggers this issue.  However, we should be using a consistent logger scheme throughout all the operators, and this should be an easy change to stop using `glog` in EdgeFS (it's only used in 1 place) and follow the same convention the other operators use.

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -1402,7 +1402,6 @@
     "github.com/ghodss/yaml",
     "github.com/go-ini/ini",
     "github.com/go-sql-driver/mysql",
-    "github.com/golang/glog",
     "github.com/google/go-cmp/cmp",
     "github.com/google/uuid",
     "github.com/icrowley/fake",

--- a/pkg/operator/edgefs/cluster/utils.go
+++ b/pkg/operator/edgefs/cluster/utils.go
@@ -21,7 +21,6 @@ import (
 	"strings"
 	"time"
 
-	"github.com/golang/glog"
 	edgefsv1alpha1 "github.com/rook/rook/pkg/apis/edgefs.rook.io/v1alpha1"
 	rookalpha "github.com/rook/rook/pkg/apis/rook.io/v1alpha2"
 	"github.com/rook/rook/pkg/operator/discover"
@@ -269,7 +268,7 @@ func (c *cluster) RemoveLabelOffNode(cs clientset.Interface, nodeName string, la
 			if !apierrs.IsConflict(err) {
 				return err
 			} else {
-				glog.V(2).Infof("Conflict when trying to remove a labels %v from %v", labelKeys, nodeName)
+				logger.Warningf("Conflict when trying to remove a labels %v from %v", labelKeys, nodeName)
 			}
 		} else {
 			break

